src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const getFilteredMovies = (searchQuery: string, movies: Movie[]) => {
+  return movies.filter(movie => {
+    const { description, title } = movie;
+    const desctiptionContaines = description.toLowerCase()
+      .includes(searchQuery.toLowerCase().trim());
+    const titleContaines = title.toLowerCase()
+      .includes(searchQuery.toLowerCase().trim());
+
+    return desctiptionContaines || titleContaines;
+  });
```

Comment:
```suggestion
  return movies.filter(({ description, title }) => {
    const desctiptionContaines = description.toLowerCase()
      .includes(searchQuery.toLowerCase().trim());
    const titleContaines = title.toLowerCase()
      .includes(searchQuery.toLowerCase().trim());

    return desctiptionContaines || titleContaines;
  });
```

Reply:
You duplicate logic on 8 and 10 lines, we probably should move this logic to separate function